pyro/contrib/funsor/handlers/plate_messenger.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, name, size=None, subsample_size=None, subsample=None, dim=Non
             name, size, subsample_size, subsample, use_cuda, device)
         self.subsample_size = subsample_size
         self._full_size = size
-        self._scale = size / subsample_size
+        self._scale = size // subsample_size
```

Comment:
Don't we want floating point division here? E.g. if my dataset has size 100 but I sample minibatches of size 40 the scale should be 2.5, not 2.0.